extensions/common/vault/vault-hashicorp/src/test/java/org/eclipse/edc/vault/hashicorp/HashicorpVaultTest.java

```diff
@@ -16,62 +16,107 @@
 
 import org.eclipse.edc.spi.monitor.Monitor;
 import org.eclipse.edc.spi.result.Result;
+import org.junit.jupiter.api.Assertions;
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
 
 import java.util.UUID;
 
 import static org.assertj.core.api.Assertions.assertThat;
-import static org.mockito.Mockito.mock;
-import static org.mockito.Mockito.times;
 import static org.mockito.Mockito.verify;
-import static org.mockito.Mockito.when;
 
 class HashicorpVaultTest {
     private static final String KEY = "key";
 
-    private HashicorpVaultClient vaultHashicorpVaultClient;
+    // mocks
+    private HashicorpVaultClient vaultClient;
     private HashicorpVault vault;
 
     @BeforeEach
     void setup() {
-        vaultHashicorpVaultClient = mock(HashicorpVaultClient.class);
-        var monitor = mock(Monitor.class);
-        vault = new HashicorpVault(vaultHashicorpVaultClient, monitor);
+        vaultClient = Mockito.mock(HashicorpVaultClient.class);
```

Comment:
static import, and I discovered that, when the type of the variable is defined, you don't have to pass the type to the `mock` method, it will understand it implicitly

Reply:
I've noticed it too, especially useful when the static type is implicit, such as for fields.